Cargo.toml

```diff
@@ -91,3 +91,4 @@ zcash_history = { git = "https://github.com/zcash/librustzcash.git", rev = "d5c5
 zcash_note_encryption = { git = "https://github.com/zcash/librustzcash.git", rev = "d5c5f048947d211c2fbef23ce986b329b91d1aa5" }
 zcash_primitives = { git = "https://github.com/zcash/librustzcash.git", rev = "d5c5f048947d211c2fbef23ce986b329b91d1aa5" }
 zcash_proofs = { git = "https://github.com/zcash/librustzcash.git", rev = "d5c5f048947d211c2fbef23ce986b329b91d1aa5" }
+orchard = { git = "https://github.com/QED-it/orchard.git", rev = "6d2834bfd3230dc6a52b5b0c99e149f92455de69"}
```

Comment:
this has changed already https://github.com/zcash/zcash/commit/5fc38dd091b7afe3961ae0cc1f9cae76bed86f14#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542L86 🤔

Reply:
The referenced changes are because we focused on (and succeeded in) ensuring that `zcashd 5.0.0` only depended on released crate versions. Adding this patch directive is the correct way for QEDIT to experiment with their patches to the `orchard` crate (and is precisely what we do for longer-lived development cycles).